### PR TITLE
fix(prow): ensures proper namespace is used in scripts

### PR DESCRIPTION
**Pull request by bartoszmajsak**
Whenever we use `kubectl` operations it should use `-n ${NAMESPACE}`, but in a few places, we were still using the `default` namespace instead. This leads to issues if you don't or can't use `default` in your cluster.

### CODE REVIEW

**Review by jwendell — APPROVED** [review]

**Comment by jwendell** [comment]
It's just waiting for the `ok-to-merge` label.

**Comment by jwendell** [comment]
You can add it, no?

**Comment by bartoszmajsak** [comment]
Is it my role to say it's ok-to-merge? I would assume it's for the reviewer to say.

I could equally well just push the merge button below instead of selecting the label :) I'm not sure what is the benefit of it. It's a UI operation after all if done by humans. Unless there's more behind `ok-to-merge` besides just merging by the bot which I'm not aware of.

**Pull request merged**
